website/public/locales/ru/message.json

```diff
@@ -18,6 +18,7 @@
   "report_placeholder": "Почему это сообщение должно быть рассмотрено?",
   "report_title": "Пожаловаться",
   "send_report": "Отправить",
+  "show_image": "Нажмите, чтобы увидеть изображение",
```

Comment:
We might need to revert this if https://github.com/LAION-AI/Open-Assistant/pull/1721 is rejected

Reply:
Okay

Reply:
@0x22almostEvil #1721 is rejected. Can you revert this change?

Reply:
yeah, sorry, was busy irl, gimme 5-10 mins